libs/platform/src/lib/components/form/form-field.ts

```diff
@@ -35,6 +35,10 @@ export abstract class FormField {
      * Indicates in which FormZone a form field should be rendered
      */
     zone: FormZone;
```

Comment:
Do we need FormZone? Do we even have zones In new design? 
The same goes to fluid. do we have such functionality now ?

Reply:
Not need more. Removed